### PR TITLE
fix(cli): Commands don't show up

### PR DESCRIPTION
**Pull request by itsdevbear**

### CODE REVIEW

**Comment by itsdevbear** [comment]
@BrickBera please verify your issue is fixed here

@calbera please verify this isn't too cooked

**Comment by itsdevbear** [comment]
good to go @calbera

File: cosmos/x/evm/types/tx.go [hunk 1]
```diff
@@ -33,6 +37,12 @@ import (
 	"pkg.berachain.dev/polaris/lib/utils"
 )
 
+var _ signing.GetSignersFunc = GetEthereumSigner
+
+func GetEthereumSigner(proto.Message) ([][]byte, error) {
+	return nil, nil
```
Comment (calbera) [inline cosmos/x/evm/types/tx.go:43]: Why empty here?

File: cosmos/testing/integration/jsonrpc/txpool_test.go [hunk 2]
```diff
@@ -50,11 +50,10 @@ var _ = Describe("Tx Pool", func() {
 		Expect(tf.Network.WaitForNextBlock()).To(Succeed())
 	})
 
-	It("should handle txpool requests: pending nonce", func() {
+	FIt("should handle txpool requests: pending nonce", func() {
```
Comment (calbera) [inline cosmos/testing/integration/jsonrpc/txpool_test.go:53]: Back to It

**Review by calbera — APPROVED** [review]

**Pull request merged**